roles/etcd/tasks/permission_directory.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Directory | check certificate permissions
+  file:
+    path: "{{ etcd_config_dir }}"
+    group: "{{ etcd_cert_group }}"
+    state: directory
+    owner: etcd
+    mode: "{{ etcd_cert_dir_mode }}"
+    recurse: yes
+  notify:
+  - Directory set var cert
+
+- name: Directory | check data permissions
+  file:
+    path: "{{ etcd_data_dir }}"
+    state: directory
+    owner: etcd
+    group: etcd
+    mode: "{{ etcd_cert_dir_mode }}"
+    recurse: yes
```

Comment:
file should not be executable

Reply:
etcd_cert_dir_mode: "0700" 
from main.yml